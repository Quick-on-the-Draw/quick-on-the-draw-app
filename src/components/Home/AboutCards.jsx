import React from 'react';
import kb from '../../assets/KB.jpg';
import nicole from '../../assets/Nicole.jpg';
import kat from '../../assets/Kat.jpg';
import './styles/AboutCards.css';

const AboutCards = () => {
    return (
        <main className='main'>
            <div className='card'>
                <h2 className='name'>Katy (KB) Boyles</h2>
                <img src={kb} alt='KB' className='bioPic' />
                <p>
                    KB is a full stack software engineer with a background in
                    Product Management in Portland, OR. She enjoys travelling
                    and dancing and generally loves to create community wherever
                    she goes.
                </p>
                <div className='links'>
                    <a
                        href='https://twitter.com/KBtrizay'
                        target='_blank'
                        rel='noreferrer'
                    >
                        Twitter
                    </a>
                    <a
                        href='https://github.com/katrinkajb'
                        target='_blank'
                        rel='noreferrer'
                    >
                        Github
                    </a>
                    <a
                        href='https://www.linkedin.com/in/katy-boyles/'
                        target='_blank'
                        rel='noreferrer'
                        className='linkedIn'
                    >
                        LinkedIn
                    </a>
                    {/* <i className='fab fa-kb fa-2x m-4 hover:bg-yellow-300'></i> */}
                </div>
            </div>
            <div className='card'>
                <h2 className='name'>Nicole Martin</h2>
                <img src={nicole} alt='Nicole' className='bioPic' />
                <p>
                    Nicole is a full stack software engineer located in
                    Portland, OR. She enjoys working with computers and
                    collaborating on fun and impactful projects that will help
                    the planet, animals, and all humans. Also a huge fan of
                    vegan food, coffee and being outside in the beautiful
                    Pacific Northwest with her dogs, Cooper and Ruby.â€™,
                </p>
                <div className='links'>
                    <a
                        href='https://twitter.com/nmartinpdx'
                        target='_blank'
                        rel='noreferrer'
                    >
                        Twitter
                    </a>
                    <a
                        href='https://github.com/nicole-m-martin'
                        target='_blank'
                        rel='noreferrer'
                    >
                        Github
                    </a>
                    <a
                        href='https://www.linkedin.com/in/nicolemartinpdx/'
                        target='_blank'
                        rel='noreferrer'
                    >
                        LinkedIn
                    </a>
                    <a href='https://www.nicolemartin.dev/'>Portfolio</a>
                </div>
            </div>
            <div className='card'>
                <h2 className='name'>Kat Sauma</h2>
                <img src={kat} alt='Kat' className='bioPic' />
                <p>
                    As an artist and a technical mind, Kat found their love of
                    graphic + ux/ui design through programming web applications.
                    They continue sharpening both ends of the tech-pencil with
                    original illustrations, choreographing pathways of art and
                    technology in performance, and creative coding with
                    Javascript libraries. Kat is a dancer and costume designer
                    and loves to cozy up with their pup Chi Chi in the Big
                    Apple.
                </p>
                <div className='links'>
                    <a
                        href='https://twitter.com/luckykatcodes'
                        target='_blank'
                        rel='noreferrer'
                    >
                        Twitter
                    </a>
                    <a
                        href='https://github.com/kat-sauma'
                        target='_blank'
                        rel='noreferrer'
                    >
                        Github
                    </a>
                    <a
                        href='https://www.linkedin.com/in/kat-sauma/'
                        target='_blank'
                        rel='noreferrer'
                    >
                        LinkedIn
                    </a>
                    <a
                        href='https://socketjockey.netlify.app/'
                        target='_blank'
                        rel='noreferrer'
                    >
                        Socket Jockey
                    </a>
                </div>
            </div>
        </main>
    );
};

export default AboutCards;
