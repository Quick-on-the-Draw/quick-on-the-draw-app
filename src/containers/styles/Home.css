@tailwind base;
@tailwind components;
@tailwind utilities;

.mainSection {
    @apply bg-scroll
    flex 
    flex-col
    justify-center
    items-center
    min-h-screen;
}

.titleImage {
    @apply h-60
    w-auto
    m-6;
}

.innerContainer {
    @apply m-6 
    p-6;
}

.header {
    @apply text-redbrown
    font-bold
    m-6;
}
