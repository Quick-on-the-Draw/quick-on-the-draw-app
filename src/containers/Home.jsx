import './styles/Home.css';
import React from 'react';
import AboutCards from '../components/Home/AboutCards';
import title from '../assets/QuickTitle.png';

const Home = () => {
    return (
        <main className='mainSection'>
            <img src={title} alt='Quick on the Draw' className='titleImage' />
            <div className='innerContainer'>
                <p>
                    Need a short break in a long day? Take your shot at a timed
                    drawing! Quick on the Draw gives a quick and fun way to take
                    your mind off of work.
                </p>
                <h3 className='headerHome'>Instructions:</h3>
                <p>
                    Click Draw to go to the Draw page. Choose your timer
                    setting, then choose your colors and brush size at the
                    bottom of the screen. Undo or Clear when you need and
                    Download your image when you're finished!
                </p>
                <h3 className='headerHome'>Technologies:</h3>
                <p>
                    Quick on the Draw was build with React.js and
                    react-canvas-draw. Tailwind.css and Craco were used for
                    styling. It is deployed on Netlify.
                </p>
            </div>
            <AboutCards />
        </main>
    );
};

export default Home;
