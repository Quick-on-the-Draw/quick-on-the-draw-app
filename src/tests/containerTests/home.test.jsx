import { render, screen } from '@testing-library/react';
import Home from '../../containers/Home';
import React from 'react';

describe.skip('home page tests', () => {
    it('looks for the navbar on the Home page', async () => {
        render(<Home />);

        const navbar = await screen.findByText('Home');
        expect(navbar).toMatchSnapshot();
    });
});
