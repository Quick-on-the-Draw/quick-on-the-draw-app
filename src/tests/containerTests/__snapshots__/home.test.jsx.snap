// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home page tests looks for the navbar on the Home page 1`] = `
Array [
  <img
    alt="Quick on the Draw"
    class="titleImage"
    src="QuickTitle.png"
  />,
]
`;
